### PR TITLE
[test] Update PrgEnv-pgi definitions on Kesch

### PR DESCRIPTION
**Pull request by ajocksch**
Also remove workaround in `gpu_direct_acc.py`.

### CODE REVIEW

**Comment by vkarak** [comment]
Please tag accordingly and put this PR in the right project and milestone.

**Comment by teojgo** [comment]
@jenkins-cscs retry kesch

**Comment by vkarak** [comment]
@teojgo @ajocksch I think CI on Kesch is expected to fail for the default `PrgEnv-pgi`.

**Comment by vkarak** [comment]
@teojgo Can you fix the conflicts? This is ready to be merged otherwise.

File: cscs-checks/prgenv/gpu_direct_acc.py [hunk 1]
```diff
@@ -18,10 +18,11 @@ def __init__(self, **kwargs):
             self.num_tasks_per_node = 1
         elif self.current_system.name in ['kesch']:
             self.modules = ['craype-accel-nvidia35']
-            self._pgi_flags = '-acc -ta=tesla:cc35 -Mnorpath'
+            self._pgi_flags = '-acc -ta=tesla:cc35'
             self.variables = {
                 'MPICH_RDMA_ENABLED_CUDA': '1',
                 'MV2_USE_CUDA': '1',
+                'G2G': '1'
                 'G2G': '1',
```
Comment (vkarak) [inline cscs-checks/prgenv/gpu_direct_acc.py]: @teojgo This does not fix the conflict; it breaks the code... I am fixing it now. You should not rely entirely on the CI to catch errors in user tests, especially after merges.

**Pull request merged**